scripts/build-glean-metadata.py

```diff
@@ -173,6 +176,12 @@ def get_app_variant_description(app):
         app = next(app for app in apps if app.app_id == app_id)
         app_is_deprecated = app.app.get("deprecated")
 
+        # app-id tags: tags specified in the annotations (and or more recent versions of an app)
+        # will always override older ones
+        for tag in app.get_tags():
+            if not app_tags.get(tag.identifier):
+                app_tags[tag.identifier] = tag.description
```

Comment:
what would be an example of this?

Reply:
Say you had the same tags specified for Firefox Android beta and Firefox Android nightly. You would want the definition for the latter to be the one we show in the Glean Dictionary, as it's more up to date:

https://github.com/mozilla/probe-scraper/blob/main/repositories.yaml#L81

Reply:
that makes sense. thank you for the example!